### PR TITLE
Update the Logstash to Logstash Native doc to reflect the multiple hosts usage.

### LINKED ISSUE #15498: [Docs] Update LS-to-LS native doc after HA support
### Description
Update `ls-ls-native.asciidoc` to reflect HA changes after releasing [`logstash-integration-logstash` HA support]( https://github.com/logstash-plugins/logstash-integration-logstash/pull/16)

### PR DESCRIPTION
**Pull request by mashhurs**
## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files (and/or docker env variables)
- [ ] I have added tests that prove my fix is effective or that my feature works

## Author's Checklist

- [ ]

## Related issues

- Closes #15498

### CODE REVIEW

File: docs/static/ls-ls-native.asciidoc [hunk 1]
```diff
@@ -36,15 +36,13 @@ input {
 
 In order to obtain the best performance when sending data from one Logstash to another, the data is batched and compressed. As such, the upstream Logstash (the sending Logstash) only needs to be concerned about configuring the receiving endpoint with these options:
 
-* `hosts` - The receiving Logstash and port. If no port specified, 9800 will be used.
-
-NOTE: In the future, {ls} will support multiple output hosts.
```
Comment (roaksoax) [inline docs/static/ls-ls-native.asciidoc:41]: Maybe we should change the note to say something like:

NOTE: Logstash supports multiple host by automatically loadbalancing events across all destination ports.

Or similar, but not a requirement.

**Review by roaksoax — APPROVED** [review]
LGTM!

**Review by roaksoax — CHANGES_REQUESTED** [review]
Actually, we need to add a section here, or a note, or something that describes how HA works.

File: docs/static/ls-ls-native.asciidoc (hunk 1 above)
Comment (roaksoax) [inline docs/static/ls-ls-native.asciidoc:41]: I think the note should highlight how we do HA, and what multiple hosts does. 

e.g. NOTE: When setting multiple hosts, Logstash will load balance the communication across the receiving endpoint. The loadbalancing strategy is [...]

**Review by roaksoax — CHANGES_REQUESTED** [review]
Generally speaking, we need to talk about what it means to do multiple hosts and how it works, generally speaking.

We also need to update all the section under  https://github.com/elastic/logstash/blob/main/docs/static/ls-ls-config.asciidoc?plain=1#L18  to reflect the fact that we will now support loadbalancing.

File: docs/static/ls-ls-native.asciidoc [hunk 2]
```diff
@@ -36,15 +36,13 @@ input {
...
+* `hosts` - The receiving Logstash host and port pairs. If no port specified, 9800 will be used.
```
Comment (mashhurs) [inline docs/static/ls-ls-native.asciidoc:39]: @roaksoax WDYT?

```suggestion
* `hosts` - The receiving one or more Logstash host and port pairs. If no port specified, 9800 will be used.

NOTE: {ls} _fairly_ distributes batched events to _all_ of configured downstream hosts. The fairness is based on, a  downstream that is running, has a capacity to receive events and no recent failures.
```

File: docs/static/ls-ls-config.asciidoc [hunk 3]
```diff
@@ -18,7 +18,7 @@ Logstash-to-Logstash communication can be achieved in one of two ways:
 This is the preferred method to implement Logstash-to-Logstash. It replaces <<ls-to-ls-http>> and has these considerations:
 
 * It relies on HTTP as the communication protocol between the Input and Output.
-* It does not provide built-in high availability. You will need to implement your own load balancer in between the Logstash output and the Logstash input.
+* It provides built-in _fairness_ load balancing, which, distributes batch of events to a downstream that is running, has a capacity to receive events and no recent failures.
```
Comment (roaksoax) [inline docs/static/ls-ls-config.asciidoc:21]: this is a bit unclear:

What does it mean "has a capacity to receive events" ? How does it know?
What about the "no recent failures" ? does this mean it is not based on that?

File: docs/static/ls-ls-config.asciidoc [hunk 4]
```diff
@@ -18,7 +18,7 @@ Logstash-to-Logstash communication can be achieved in one of two ways:
...
 * If you need a proxy between the Logstash instances, you can use any HTTP proxy.
```
Comment (roaksoax) [inline docs/static/ls-ls-config.asciidoc:22]: We should probably remove this line too.

File: docs/static/ls-ls-native.asciidoc (hunk 2 above)
Comment (roaksoax) [reply docs/static/ls-ls-native.asciidoc:39]: Let's pick one place where we put the explanation. Maybe we should keep it here, and not on the other part.

File: docs/static/ls-ls-config.asciidoc (hunk 3 above)
Comment (roaksoax) [reply docs/static/ls-ls-config.asciidoc:21]: Also, I think we can simply summarize that we support loadbalancing when providing multiple outputs. And the explanation of how we distribute the load should be present in the other page, as per below comment.

**Review by roaksoax — CHANGES_REQUESTED** [review]

File: docs/static/ls-ls-config.asciidoc (hunk 3 above)
Comment (roaksoax) [inline docs/static/ls-ls-config.asciidoc:21]:
```suggestion
* It supports multiple hosts, providing high availability by load balancing equally amongst the multiple destination hosts.
```

**Review by roaksoax — APPROVED** [review]
LGtm!

File: docs/static/ls-ls-config.asciidoc (hunk 4 above)
Comment (mashhurs) [inline docs/static/ls-ls-config.asciidoc:22]:
```suggestion
```

**Pull request merged**